pkg/config/generate.go

```diff
@@ -328,8 +328,13 @@ func Generate(cs *acsapi.OpenShiftManagedCluster) (err error) {
 	}
 
 	if len(c.HtPasswd) == 0 {
-		if c.HtPasswd, err = makeHtPasswd("demo", "demo"); err != nil {
-			return
+		pass, err := randomString(10)
+		c.HtPasswdPlain = pass
+		if err != nil {
+			return err
+		}
+		if c.HtPasswd, err = makeHtPasswd("osadmin", pass); err != nil {
```

Comment:
Open follow-up to make this a derived config

Reply:
Cancel that - the hash will change each time - it can't be derived

Reply:
We just need to get it each time. Kinda password rotation? Prevents people from storing passwords for clusters?